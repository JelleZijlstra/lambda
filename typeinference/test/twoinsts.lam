let f = \x. x.bar in
(f {bar = 3}, (f {bar = true, baz = 4}, f {bar = \x. x, foo = ()}))
