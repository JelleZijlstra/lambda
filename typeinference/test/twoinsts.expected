%%stdout%%
Result: module
	let __result__ = (3, (true, \x. x))
end
%%stderr%%
